admin/schema.php

```diff
@@ -865,6 +865,10 @@
 # Enable gravatar plugin if avatars are enabled
 $g_upgrade[205] = array( 'UpdateFunction', 'gravatar_plugin', array() );
 
+# Add description field to custom field definition
+$g_upgrade[206] = array( 'AddColumnSQL', array( db_get_table( 'custom_field' ), "
+	description					XL		NULL DEFAULT NULL" ) );
```

Comment:
Do we want to support long descriptions here?  Should we use a one line for description?  That would reflect on the db schema and UI.